290.py

```diff
@@ -0,0 +1,19 @@
+class Solution:
+    def wordPattern(self, pattern: str, s: str) -> bool:
+        strs = s.split()
```

Comment:
`strs` is not descriptive. An abbreviated variable name like `strs` doesn't fit with a name like `pattern` in the same function. Let's use something like `words` for consistency.